docs(common)troubleshooting - add Failed to find error (#2053)
telerik/blazor-docs: knowledge-base/common-failed-to-find.md

---

> title: Failed To Find a Valid Digest in the 'Integrity' Attribute

Reviewer: Link to this KB from the General Troubleshooting article https://docs.telerik.com/blazor-ui/troubleshooting/general-issues

> title: Failed To Find a Valid Digest in the 'Integrity' Attribute for Resource ... With Computed SHA-256 Integrity

Reviewer 2: Try to limit the length of the titles. See https://docs.telerik.com/style-guide/titles-and-headings#general.
```suggestion
title: Failed To Find a Valid Digest in the 'Integrity' Attribute
```

description: Learn what causes the Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ... error and how to fix it.
type: troubleshooting

> page_title: How to Fix Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...

Reviewer 2: ```suggestion
page_title: Failed To Find a Valid Digest in the 'Integrity' Attribute
```

slug: common-kb-failed-to-find
tags: telerik, blazor, troubleshooting, webassembly
ticketid: 1645828, 1495720, 1540839, 1557096, 1557819, 1587286, 1615743, 1617356, 1636012
res_type: kb
---

## Environment

<table>
    <tbody>
        <tr>
            <td>Product</td>
            <td>UI for Blazor</td>
        </tr>
    </tbody>
</table>

## Description

> I get the `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` error after upgrading Telerik.UI.forBlazor Nuget package in my Blazor WebAssembly and publishing the app.

Reviewer: ```suggestion
I get a `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` error after upgrading the `Telerik.UI.for.Blazor` NuGet package and publishing my Blazor WebAssembly app.
```

> This knowledge base article deals with the `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` error.

Reviewer 2: In the [description](https://docs.telerik.com/style-guide/kb-articles#troubleshooting-scenarios), try to state the question or describe the context of the error message. For example:
* When does it appear?
* What was the customer doing before the error was thrown?

You could try something like this:
* When I [describe what the customer did or could have done], the application started throwing ... errors in the browser's console.
* When the application [describe the trigger], I started seeing ... errors in ... .

## Error Message

>warning Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ... The resource has been blocked.
>
> Unknown error occured while trying to verify integrity.

## Cause

> The error `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` is a general Blazor WebAssembly issue, which can result from the build process, publishing process, or browser cache. The Telerik UI for Blazor components are not the cause.

Reviewer: Use a similar amount of "exception" and "error"

> The error `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` is a general Blazor WebAssembly issue, which can result from the build process, publishing process, or browser cache. The Telerik UI for Blazor components are not the cause.

Reviewer: ```suggestion
The error `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` is a general Blazor WebAssembly issue, which can result from the build process, publishing process, or browser cache. The Telerik UI for Blazor components are not directly related to the problem.
```

> The error `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` is a general Blazor WebAssembly issue, which can result from the build process, publish process or browser cache. The Telerik UI for Blazor components are not the cause.

Reviewer 2: ```suggestion
The error `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` is a general Blazor WebAssembly issue, which can result from the build process, publishing process, or browser cache. The Telerik UI for Blazor components are not the cause.
```

## Solution

> It's hard to provide a specific advice, but here are some online documentation and discussions on the matter, which suggest possible solutions:

Reviewer: ```suggestion
The fix for the exception can vary, depending on the cause. Here are some online documentation and discussions on the matter, which suggest possible solutions:
```

type: troubleshooting
page_title: Failed To Find a Valid Digest in the 'Integrity' Attribute
slug: common-kb-failed-to-find
tags: telerik, blazor, troubleshooting, webassembly
ticketid: 1645828, 1495720, 1540839, 1557096, 1557819, 1587286, 1615743, 1617356, 1636012
res_type: kb
---

## Environment

<table>
    <tbody>
        <tr>
            <td>Product</td>
            <td>UI for Blazor</td>
        </tr>
    </tbody>
</table>

## Description

I get a `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` error after upgrading the `Telerik.UI.for.Blazor` NuGet package and publishing my Blazor WebAssembly app.

## Error Message

>warning Failed to find a valid digest in the 'integrity' attribute for resource 'https://.../_framework/Telerik.Recurrence.dll' with computed SHA-256 integrity ... The resource has been blocked.
>
> Unknown error occured while trying to verify integrity.
>
>Error: Failed to start platform. Reason: TypeError: Failed to fetch

Other possible resources that may be referred to in the error:

> * https://.../_framework/Telerik.Documents.SpreadsheetStreaming.dll
> * https://.../_framework/Telerik.DataSource.dll
> * https://.../_framework/Telerik.Zip.dll
> * https://.../_framework/Telerik.Blazor.dll
> * https://.../_framework/Telerik.FontIcons.dll
> * https://.../_framework/Telerik.SvgIcons.dll

Reviewer: ```suggestion
* Telerik.Documents.SpreadsheetStreaming.dll
* Telerik.DataSource.dll
* Telerik.Zip.dll
* Telerik.Blazor.dll
* Telerik.FontIcons.dll
* Telerik.SvgIcons.dll
```
